ledger/participant-state/kvutils/app/src/main/scala/com/daml/ledger/participant/state/kvutils/app/Config.scala

```diff
@@ -390,13 +390,21 @@ object Config {
           .text(
             s"Number of events fetched from the index for every round trip when serving streaming calls. Default is ${IndexConfiguration.DefaultEventsPageSize}."
           )
+          .validate(pageSize =>
```

Comment:
I would disallow petty pages, i.e. should be >=100

Reply:
I would agree as well, but this would be a breaking change, don't you think? What if there already are applications running with page size 50?